Resources/Prototypes/Corvax/Entities/Objects/Materials/Sheets/other.yml

```
@@ -0,0 +1,32 @@
+  id: SheetPrinter
+  suffix: Full
+  components:
+  - type: Material
+  - type: PhysicalComposition
+    materialComposition:
+      SheetPrinter: 100
+  - type: Stack
+    stackType: SheetPrinter
+    baseLayer: base
+    layerStates:
+    - paper
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)